lib/logstash/string_interpolation.rb

```diff
@@ -108,7 +108,7 @@ def evaluate(event)
 
       case value
       when nil
-        "%{#{@key}}"
+        "%{#{@key}}".encode(Encoding::UTF_8)
```

Comment:
This would only not be UTF-8, I think, if the top of this file doesn't have `# encoding: utf-8`, and I see it does not. That'll be the fix :)

We should ensure all our source code has '# encoding: utf-8` at the top of it.